packages/now-go/index.ts

```diff
@@ -80,12 +91,8 @@ Learn more: https://github.com/golang/go/wiki/Modules
   ]);
 
   const srcPath = join(goPath, 'src', 'lambda');
-  let downloadedFiles;
-  if (meta.isDev) {
-    downloadedFiles = await download(files, workPath, meta);
-  } else {
-    downloadedFiles = await download(files, srcPath);
-  }
+  const downloadPath = meta.isDev ? workPath : srcPath;
+  let downloadedFiles = await download(files, downloadPath, meta);
```

Comment:
```suggestion
  const downloadedFiles = await download(files, downloadPath, meta);
```

Reply:
@TooTallNate It can't be `const` because it is reassigned below (also lint would have complained 😉 )